src/frontend/src/components/ReactHookEditableList.tsx

```diff
@@ -1,36 +1,58 @@
-import { Grid, Button, IconButton, TextField } from '@mui/material';
+import { Grid, IconButton, TextField, InputLabel } from '@mui/material';
 import DeleteIcon from '@mui/icons-material/Delete';
 import { FieldArrayWithId, UseFieldArrayRemove, UseFormRegister, UseFieldArrayAppend } from 'react-hook-form';
+import { Box } from '@mui/system';
+import { NERButton } from './NERButton';
 
 interface ReactHookEditableListProps {
   name: string;
   ls: FieldArrayWithId[];
   register: UseFormRegister<any>;
   append: UseFieldArrayAppend<any, any>;
   remove: UseFieldArrayRemove;
+  bulletName?: string;
 }
 
-const ReactHookEditableList: React.FC<ReactHookEditableListProps> = ({ name, ls, register, append, remove }) => {
+const ReactHookEditableList: React.FC<ReactHookEditableListProps> = ({ name, ls, bulletName, register, append, remove }) => {
   return (
     <>
-      {ls.map((_element, i) => {
-        return (
-          <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
-            <TextField required autoComplete="off" sx={{ width: 9 / 10 }} {...register(`${name}.${i}.detail`)} />
-            <IconButton type="button" onClick={() => remove(i)} sx={{ mx: 1, my: 0 }}>
-              <DeleteIcon />
-            </IconButton>
-          </Grid>
-        );
-      })}
-      <Button
+      <Grid container columnSpacing={{ xs: 4, sm: 2, md: 3 }}>
+        {ls.map((_element, i) => {
+          return (
+            <Grid item xs={12} md={6} lg={4} key={`${name}-${i}`}>
+              <InputLabel sx={{ marginBottom: '5px' }}>
+                {bulletName} {i + 1}
+              </InputLabel>
+              <Box sx={{ display: 'flex', alignItems: 'start', marginBottom: '20px' }}>
+                <TextField
+                  required
+                  autoComplete="off"
+                  placeholder={'Enter ' + bulletName + ' . . .'}
+                  sx={{ width: 12 / 12 }}
+                  multiline
+                  maxRows={3}
+                  {...register(`${name}.${i}.detail`)}
+                />
+                <IconButton
+                  type="button"
+                  onClick={() => remove(i)}
+                  sx={{ mx: 1, color: 'red', marginLeft: '15px', borderRadius: '4px', outline: 'solid' }}
```

Comment:
can you add a marginTop: '3px' to this sx so that the button is more inline with the textbox